partner_identification/models/res_partner_id_category.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# © 2004-2010 Tiny SPRL http://tiny.be
+# © 2010-2012 ChriCar Beteiligungs- und Beratungs- GmbH
+#             http://www.camptocamp.at
+# © 2015 Antiun Ingenieria, SL (Madrid, Spain)
+#        http://www.antiun.com
+#        Antonio Espinosa <antonioea@antiun.com>
+# ©  2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import api, models, fields
+from openerp.exceptions import ValidationError, UserError
+from openerp.tools.safe_eval import safe_eval
+from openerp.tools.translate import _
+
+
+class ResPartnerIdCategory(models.Model):
+    _name = "res.partner.id_category"
+    _order = "name"
+
+    code = fields.Char(string="Code", size=16, required=True)
+    name = fields.Char(string="ID name", required=True, translate=True)
+    active = fields.Boolean(string="Active", default=True)
+    validation_code = fields.Text(
+        'Python validation code',
+        help="Python code called to validate an id number.",
+        default="""
+# Python code. Use failed = True to specify that the id number is not valid.
```

Comment:
You can achieve the same formatting with single quotes:
```
    validation_code = fields.Text(
        'Python validation code',
        help="Python code called to validate an id number.",
        default="\n# Python code. Use failed = True to specify that the id "
                "number is not valid.\n"
                "# You can use the following variables :\n"
                "#  - self: browse_record of the current ID Category "
                "browse_record\n"
                "#  - id_number: browse_record of ID number to validate")
```
Even better, put it on a method and make this translatable:
```
    def _default_validation_code(self):
        return _("\n# Python code. Use failed = True to specify that the id "
                 "number is not valid.\n"
                 "# You can use the following variables :\n"
                 "#  - self: browse_record of the current ID Category "
                 "browse_record\n"
                 "#  - id_number: browse_record of ID number to validate")
```